relay-event-derive/src/lib.rs

```diff
@@ -595,6 +605,17 @@ fn parse_field_attributes(
                                         panic!("Got non string literal for retain");
                                     }
                                 }
+                            } else if ident == "trim" {
+                                match name_value.lit {
+                                    Lit::Str(litstr) => match litstr.value().as_str() {
```

Comment:
Could make it a 'real' boolean, `trim = true` and use `Lit::Bool` here.

Reply:
That would be better, but I want to stay consistent with `required = "true"` and `pii = "true"`.